hnn_core/viz.py

```diff
@@ -242,29 +244,45 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
         The matplotlib figure handle.
     """
     import matplotlib.pyplot as plt
-    from .dipole import Dipole
+    from .dipole import Dipole, average_dipoles
 
     if ax is None:
         _, ax = plt.subplots(1, 1, constrained_layout=True)
 
     if isinstance(dpl, Dipole):
         dpl = [dpl]
+        average = False
```

Comment:
```suggestion
```

Now with `average = False` as default you don't need to override the user-provided setting. Also, this avoids the different behavior between `dpl` and `[dpl]`. Both should give the same plot in my opinion.